website/docs/examples/filter_client/README.md

```diff
@@ -86,28 +86,29 @@ By default, the host address is assumed to be `localhost`.
 
 ## 4. Customize the run
 
-Let's have some fun and stream the state to your laptop over the Wifi,
-using the gRPC client.
-
-:::tip
-You need to discover the WiFi address of your Amiga Brain using
-the `WifiClient` (coming soon)
-:::
-
-```bash
-python3 main.py --help
-
-# usage: amiga-filter-stream [-h] --service-config SERVICE_CONFIG
-
-# optional arguments:
-#   -h, --help            show this help message and exit
-#   --service-config SERVICE_CONFIG
-#                         The filter service config.
+To run this script from your PC, you need to update the `service_config.json`
+by modifying the `host` field with your Amiga brain name.
+Please check out the [Amiga Development 101](../../concepts/system_overview/README.md#where-to-run-the-examples)
+for more details.
+
+```json
+{
+    "name": "filter",
+    "port": 20001,
+    "host": "localhost",
+    "log_level": "INFO",
+    "subscriptions": [
+        {
+            "uri": {
+                "path": "/state",
+                "query": "service_name=filter"
+            },
+            "every_n": 1
+        }
+    ]
+}
```

Comment:
Suggest to remove this from all the examples and have an example of changing the `host` in the single source of truth

* Suggestion is missing removing the final backticks, so make sure not to commit directly
```suggestion
```